wn/_download.py

```diff
@@ -98,7 +98,7 @@ def _download(urls: Sequence[str], progress: ProgressHandler) -> Path:
             try:
                 with open(path, 'wb') as f:
                     progress.set(status='Requesting', count=0)
-                    with httpx.stream("GET", url, timeout=TIMEOUT) as response:
+                    with httpx.stream("GET", url, timeout=TIMEOUT, follow_redirects=True) as response:
```

Comment:
This will lead to a formatting warning as the line is now too long.

Rather than just reformatting, I think this is a good opportunity to use a proper httpx client:

```python
def _download(urls: Sequence[str], progress: ProgressHandler) -> Path:
    try:
        client = httpx.Client(timeout=TIMEOUT, follow_redirects=True)
        for i, url in enumerate(urls, 1):
            ...
```

Then...

```python
                    with client.stream("GET", url) as response:
                        ...
```

Then...

```python
    except Exception:
        path.unlink(missing_ok=True)
        raise
    finally:
        client.close()
```

Reply:
This should also make things more efficient when downloading multiple resources (e.g., OMW).

Reply:
I went ahead and made the change since you allowed maintainers to make edits.